### PR TITLE
Fix `grunt test:reference`

### PR DESCRIPTION
**Pull request by ronyeh**
Fix https://github.com/vexflow/vexflow/issues/147
Also adds Sebastian and Leipzig as test fonts.

### CODE REVIEW

File: tools/generate_images_jsdom.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ registerFont(path.join(fontsDir, 'bravura/bravura.otf'), { family: 'Bravura' });
 registerFont(path.join(fontsDir, 'academico/academico.otf'), { family: 'Academico' });
 registerFont(path.join(fontsDir, 'petaluma/petaluma.otf'), { family: 'Petaluma' });
 registerFont(path.join(fontsDir, 'petalumascript/petalumascript.otf'), { family: 'Petaluma Script' });
+registerFont(path.join(fontsDir, 'sebastian/sebastian.otf'), { family: 'Sebastian' });
 registerFont(path.join(fontsDir, 'gonvillesmufl/gonvillesmufl.otf'), { family: 'Gonville' });
 registerFont(path.join(fontsDir, 'finaleash/finaleash.otf'), { family: 'Finale Ash' });
```
Comment (rvilarl) [inline tools/generate_images_jsdom.js:24]: The list is not complete, Leipzig is included in tests but not here (probably to reduce the required time of processing), but sebastian associated text font (Nepomuk) is also not included.

**Comment by rvilarl** [comment]
I will merge to fix the testing. If the comment above is relevant we can make another PR.

File: tools/generate_images_jsdom.js (hunk 1 above)
Comment (ronyeh) [reply tools/generate_images_jsdom.js:24]: This script is for jsdom image generator which we do not use actively. We mostly use the puppeteer version. I didn't have time to test all the edge cases since this was a quick fix. 

We probably need to go through the test helpers and puppeteer + Jason scripts to organize it and maybe even optimize it, since we are testing a whole lot of fonts now.

Comment (rvilarl) [reply tools/generate_images_jsdom.js:24]: Agree it  should be possible to select only one and reduce time. Most of the times I only look at Bravura differences.
